app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -63,6 +67,27 @@ def selfie_status
           :not_processed
         end
 
+        def verification_response_data
+          return {
+            customer_profile: get_data(DATA_PATHS[:customer_profile]),
+            reference_id: get_data(DATA_PATHS[:reference_id]),
+            reason_codes: get_data(DATA_PATHS[:reason_codes]),
+            document_type: get_data(DATA_PATHS[:document_type]),
+            decision: get_data(DATA_PATHS[:decision]),
+            state: state,
+            state_id_type: state_id_type,
+            flow_path: nil,
+            liveness_checking_required: @biometric_comparison_required,
+            issue_year: get_year(state_id_issued),
+            doc_auth_success: successful_result?,
+            vendor: 'Socure',
+            address_line2_present: address2.present?,
+            zip_code: zipcode,
+            birth_year: dob&.year,
+            liveness_enabled: @biometric_comparison_required,
+          }
+        end
+
```

Comment:
instead of creating this method ... can we use the existing extra_attributes_method  for all of the attributes alredy in this file and them merge in { vendor_request_time_in_ms: } when making the calls to analytics ... i think it would be moreso inline the pattern of how we are currently handling FormResponses